cache/blobs.go

```diff
@@ -62,6 +63,8 @@ func (sr *immutableRef) computeBlobChain(ctx context.Context, createIfNeeded boo
 }
 
 func computeBlobChain(ctx context.Context, sr *immutableRef, createIfNeeded bool, comp compression.Config, s session.Group, filter map[string]struct{}) error {
+	sourceDateEpoch := epoch.FromContext(ctx)
```

Comment:
I don't really like this hiding function parameters into context. If containerd has decided this for its functions we should at least keep our callers clean.

Reply:
Updated not to use ctx